app/models/propertyrepresentation/AppointNewAgentSession.scala

```diff
@@ -16,15 +16,78 @@
 
 package models.propertyrepresentation
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{JsError, JsResult, JsValue, Json, OFormat, Reads, __}
 
-case class AppointNewAgentSession(
+sealed trait AppointNewAgentSession {
+  val status: String
+}
+
+case class SearchedAgent(
+      agentCode: Long,
+      agentOrganisationName: String,
+      agentAddress: String,
+      status: String = "SearchedAgent")
```

Comment:
why are we not using Enumerations for these?

Reply:
now using enums